api/v1alpha1/flowcollector_types.go

```diff
@@ -300,9 +300,15 @@ type FlowCollectorLoki struct {
 	//+kubebuilder:validation:optional
 	// QuerierURL specifies the address of the Loki querier service, in case it is different from the
 	// Loki ingester URL. If empty, the URL value will be used (assuming that the Loki ingester
-	// and querier are int he same host).
+	// and querier are in the same host).
```

Comment:
nitpicking, but "same host" is not totally correct IMO, I would rephrase as "same server", no? (they can be on same host (node), but different servers / deployments)